### PR TITLE
Added a button to download vulnerability in COSV

### LINKED ISSUE #2602: Allow to download COSV file

### PR DESCRIPTION
**Pull request by nulls**
### What's done:
- added a new method in CosvRepository to download COSV as a stream
- added an endpoint to download vulnerability as COSV
- added a button on FE

It closes #2602

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt [hunk 1]
```diff
@@ -247,6 +249,18 @@ internal val headerMenu: FC<HeaderMenuProps> = FC { props ->
                                 props.setIsTableView { !it }
                             }
                         }
+                        if (props.selectedMenu == VulnerabilityTab.INFO) {
+                            a {
```
Comment (Cheshiriks) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt:253]: I think ```Link { }``` would be preferable here

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt (hunk 1 above)
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt:253]: looks like `Link` is a link to some page on site.
I tried it and it redirect to main page and that all

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt:253]: Oh, I didn’t notice that you have a file download here. OK

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt:253]: found example with full url, will try to again

Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt:253]: tried with `Link {}` and relative path -- doesn't work too

**Pull request merged**